src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -25,11 +25,15 @@ class ResizeFormListener implements EventSubscriberInterface
 {
     /**
      * @var string
+     *
+     * @deprecated since version 2.7, will be replaced by a $entryType private property in 3.0.
```

Comment:
I don't think you should rename the properties. Otherwise you'd have to add the new properties already and show deprecation notes when people use the old ones. I don't think the effort required to do this is worth it.